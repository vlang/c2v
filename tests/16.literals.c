int main() {
  char c1 = '\\';
  char c2 = '`';
  char c3 = '.';
}