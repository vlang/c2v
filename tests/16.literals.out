[translated]
module main

fn main() {
	c1 := `\\`
	c2 := `\``
	c3 := `.`
}
